src/master/jumpstart.md

```diff
@@ -35,6 +35,9 @@ SDKs make it comfortable to talk to the server from your favourite language, wit
 The most well-known immudb SDK is written in [Golang](https://golang.org/), but there are SDKs available for Python, NodeJS, Java and others.
 
 For other unsupported programming languages, [immugw](/master/immugw/) provides a REST gateway that can be used to talk to the server via generic HTTP.
+ 
+The [Ruby](https://github.com/ankane/immudb-ruby) SDK is maintained by the community, credits to [Ankane](https://github.com/ankane).
```

Comment:
We should have some table with SDKs for programming languages with links to those SDK docs (with exception of golang SDK where it is documented on docs.immudb.io), most likely we would split this into two parts where officially supported SDKs are listed first followed by community maintained ones.

Reply:
To me it does not make senso to split right now as we just have one SDK maintainer by the community. I added a column make it clear